src/Cooked/MockChain/Testing.hs

```diff
@@ -279,6 +114,9 @@ instance IsProp Bool where
   testConjoin = and
   testDisjoin = or
 
+-- * Extra HUnit assertions
```

Comment:
What are those exactly. Do we ever need them?

Reply:
They are used in some testing modules. They used to be in some sort of a utility module in tests which was removed a while ago.

Reply:
Ok. What do you think about those?

Reply:
I think they are here for a lack of a better location, but they do not create any harm.